cmd/earthly/main.go

```diff
@@ -2492,21 +2495,28 @@ func (app *earthlyApp) actionAccountLogin(c *cli.Context) error {
 		if err != nil {
 			return err
 		}
-		fmt.Printf("Logged in as %q using token auth\n", email) // TODO display if using read-only token
+		app.console.Printf("Logged in as %q using token auth\n", email) // TODO display if using read-only token
+		app.printLogSharingMessage()
 	} else {
 		err = cc.SetPasswordCredentials(email, string(pass))
 		if err != nil {
 			return err
 		}
-		fmt.Printf("Logged in as %q using password auth\n", email)
-		fmt.Printf("Warning unencrypted password has been stored under ~/.earthly/auth.credentials; consider using ssh-based auth to prevent this.\n")
+		app.console.Printf("Logged in as %q using password auth\n", email)
+		app.console.Printf("Warning unencrypted password has been stored under ~/.earthly/auth.credentials; consider using ssh-based auth to prevent this.\n")
+		app.printLogSharingMessage()
 	}
 	if err = cc.Authenticate(); err != nil {
 		return errors.Wrap(err, "authentication with cloud server failed")
 	}
 	return nil
 }
 
+func (app *earthlyApp) printLogSharingMessage() {
+	app.console.Printf("Log sharing is enabled by default. If you would like to disable it, run:")
```

Comment:
nit: move `\n` from below printf up to this line.

Reply:
I smooshed all the printing together with the intent on making the resulting layout/formatting more explicit.